libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -133,6 +139,7 @@ class AP_MotorsHeli_RSC {
     AP_Int8         _runup_time;              // Time in seconds for the main rotor to reach full speed.  Must be longer than _rsc_ramp_time
     AP_Int16        _critical_speed;          // Rotor speed below which flight is not possible
     AP_Int16        _idle_output;             // Rotor control output while at idle
+    AP_Int16        _ext_gov_arot_sp;          // Microsecond signal value sent to external governor when in autorotation
```

Comment:
"Microsecond signal" means "PWM"?  If "yes" let's stick to PWM .. sorry to be picky but I personally prefer if we don't introduce new words unless we really need to 'cuz it confuses people who aren't really familiar with the code.